secure-storage-sample/index.js

```diff
@@ -0,0 +1,20 @@
+const secureStorage = require("uxp").storage.secureStorage;
+
+async function saveSecureKey() {
+    const userInput = document.getElementById("secure").value;
+    await secureStorage.setItem("secure", userInput);
+    document.getElementById("secure").value = "";

```

Comment:
Small ask, but can you make the indentation level consistent with the rest of the example?